x-pack/plugins/fleet/server/services/output.ts

```diff
@@ -17,6 +18,30 @@ const SAVED_OBJECT_TYPE = OUTPUT_SAVED_OBJECT_TYPE;
 
 const DEFAULT_ES_HOSTS = ['http://localhost:9200'];
 
+// differentiate
+function isUUID(val: string) {
+  return (
+    typeof val === 'string' &&
+    val.match(/[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}/)
+  );
+}
+
+export function outputIdToUuid(id: string) {
```

Comment:
Encrypted saved object only work with an id in a uuid format, as we plan to use encrypted saved object for output to store fleet server service token,  to avoid migrating id  I think it's good idea to keep that even if fleet server service token do not land in 7.16.0